devicehive-backend/src/main/resources/messages.properties

```diff
@@ -108,4 +108,5 @@ NO_NETWORKS_ASSIGNED_TO_USER=User has no networks assigned to him
 CANT_DELETE_CURRENT_USER_KEY=You can not delete a user or access key that you use to authenticate this request
 CANT_DELETE_LAST_DEFAULT_ACCESS_KEY=You can not delete your last default access key
 PAYLOAD_NOT_FOUND=Missing or invalid payload in request
-SUBSCRIPTION_NOT_FOUND=Subscription with id %s was not found
\ No newline at end of file
+SUBSCRIPTION_NOT_FOUND=Subscription with id %s was not found
+DEVICE_ID_CANNOT_CONTAIN_COMMAS=Device Id cannot contain commas
```

Comment:
It seems we should avoid any special characters except dash ("-") as dash is used in uuid